lang/CMakeLists.txt

```diff
@@ -9,7 +9,13 @@ file(GLOB _languages *.po)
 set(_mofiles)
 foreach(_pofile ${_languages})
 	get_filename_component(_language ${_pofile} NAME_WE)
-	set(_mofile ${CMAKE_CURRENT_BINARY_DIR}/${_language}.mo)
+	if(WIN32)
+		# For some reason we need this structure on Windows to make localization work for a local build folder without install.
+		set(_mofile ${CMAKE_CURRENT_BINARY_DIR}/${_language}/LC_MESSAGES/${CMAKE_PROJECT_NAME}.mo)
```

Comment:
then why not using the same tree on linux as well?

Reply:
Because msgfmt can't output to directories that don't exist whilst windows can !?

And if i create the directory beforehand with an extra command, that doesn't work on windows since the first output (the created folder) is set to `_mofile` which breaks msgfmt again >.>"

This way it works with the least amount of changes 😂 gotta have fun with cross platformness...

Reply:
ok